cleancat/base.py

```diff
@@ -913,6 +914,70 @@ def clean(self, value):
         )
 
 
+class UUID(String):
+    """Schema field for UUIDs.
+
+    It converts to (and from) python UUID objects.
```

Comment:
This isn't super clear at a glance – I think this would be a bit easier (albeit more verbose):

```
    It handles deserialization from a string to a Python UUID object
    and serialization from a Python UUID object to a string.
```